src/sentry/static/sentry/app/views/usageStats/usageStatsOrg.tsx

```diff
@@ -0,0 +1,248 @@
+        value: accepted,
+      },
+      {
+        title: t('Filtered'),
+        description: tct(
+          'Filtered [dataCategory] were blocked due to your inbound data filter rules',
+          {dataCategory}
+        ),
+        value: filtered,
+      },
+      {
+        title: t('Dropped'),
+        description: tct(
+          'Dropped [dataCategory] were discarded due to rate-limits, quota limits, or spike protection',
+          {dataCategory}
+        ),
+        value: dropped,
+      },
+    ];
+
+    return (
+      <CardWrapper>
+        {cardData.map((c, i) => (
+          <StyledCard key={i}>
+            <HeaderTitle>
+              <OverflowEllipsis>{c.title}</OverflowEllipsis>
+              {c.description && (
+                <QuestionTooltip size="sm" position="top" title={c.description} />
+              )}
+            </HeaderTitle>
+            <CardContent>
+              <OverflowEllipsis>{c.value}</OverflowEllipsis>
+            </CardContent>
+          </StyledCard>
+        ))}
+      </CardWrapper>
+    );
+  }
+
+  renderChart(e?: Error) {
+    // TODO(leedongwei): Poke someone for a error-state design
+    if (this.state.error || e) {
+      return (
+        <Panel>
+          <PanelBody>
+            <p>UsageStatsOrganization has an error: {e?.message}</p>
+          </PanelBody>
+        </Panel>
+      );
+    }
+
+    if (this.state.loading || !this.state.orgStats) {
+      return (
+        <Panel>
+          <PanelBody>
+            <LoadingIndicator />
+          </PanelBody>
+        </Panel>
+      );
+    }
+
+    const {dataCategory} = this.state;
+    const {onChangeDataCategory} = this.props;
+    const {stats} = this.formattedOrgStats;
+
+    const today = moment().format('YYYY-MM-DD');
+    const start = moment().subtract(30, 'days').format('YYYY-MM-DD');
+
+    return (
+      <Panel>
+        <Chart
+          hasTransactions
+          hasAttachments={false}
+          usagePeriodStart={start}
+          usagePeriodEnd={today}
+          usagePeriodToday={today}
+          statsAttachments={stats}
+          statsErrors={stats}
+          statsTransactions={stats}
+        />
+
+        <ButtonBar active={dataCategory} merged>
+          {Object.keys(DataCategory).map(k => {
+            return (
+              <Button
+                key={DataCategory[k]}
+                barId={DataCategory[k]}
+                onClick={() => onChangeDataCategory(DataCategory[k])}
+              >
+                {capitalize(DataCategory[k])}
+              </Button>
+            );
+          })}
+        </ButtonBar>
+      </Panel>
+    );
+  }
+
+  renderLoading() {
+    return (
+      <React.Fragment>
+        {this.renderCards()}
+        {this.renderChart()}
+      </React.Fragment>
+    );
+  }
+
+  renderError(e: Error) {
+    return (
+      <React.Fragment>
+        {this.renderCards()}
+        {this.renderChart(e)}
+      </React.Fragment>
+    );
+  }
+
+  renderBody() {
```

Comment:
You could override `renderComponent` as your loading, error and done states have the same logic.